geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1929,6 +1929,17 @@
    * Geode 1.0
    */
   String DISTRIBUTED_TRANSACTIONS = "distributed-transactions";
+  /**
+   * The static String definition of the <i>"ssl-endpoint-identification-enabled"</i> property <a
+   * name="ssl-endpoint-identification-enabled"/a>
+   * </p>
+   * <U>Description</U>: If true, the endpoint identification will be enabled during SSL/TLS
+   * handshake.
+   * </p>
+   * <U>Default</U>: code>"false"</code>
+   * <U>Since</U>: Geode 1.8
+   */
+  String SSL_ENDPOINT_IDENTIFICATION_ENABLED = "ssl-enable-endpoint-identification";
```

Comment:
Can you add some more details to the description of this? Like how it interacts with truststores and keystores, which which connections this affects and why you would want to turn it on?

Reply:
I have added details about hostname verification. This does not change the current interactions with truststore and keystore.